examples/graphene_apps/python_worker/avalon_worker/worker_signing.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python3
+
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import logging
+import base64
+from ecdsa import SigningKey, VerifyingKey, SECP256k1
+from ecdsa.util import sigencode_der, sigdecode_der
+import avalon_worker.worker_hash as worker_hash
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler(sys.stdout))
+
+# -------------------------------------------------------------------------
+
+
+class WorkerSign():
```

Comment:
Is there a reason for not using common/crypto_utils/avalon_crypto_utils/signature.py?
This file duplicates the functionality.

Reply:
The plan was to avoid Avalon build dependencies on Python worker and keep it standalone for now. As of now we can build graphene python worker without any dependency on Avalon modules. Avalon crypto utils will be restructured in next release. This PR introduces 2 new crypto related classes - worker_encryption.py and worker_signing.py. Another option is to modify existing Avalon Crypto utils to use these classes. But it was too late/risky to modify existing Avalon crypto utils at this stage of release.  I'll create a ReadMe for Graphene Worker and include above assumptions.

Reply:
@manojgop Thanks. A README to memorialize the reasons for the new crypto code would be good.